components/brave_wallet_ui/utils/buy-asset-url.ts

```diff
@@ -1,21 +1,20 @@
-import { Network, AccountAssetOptionType, UserAccountType } from '../constants/types'
-import { NetworkOptions } from '../options/network-options'
+import { AccountAssetOptionType, UserAccountType } from '../constants/types'
 
 const wyreID = 'AC_MGNVBGHPA9T'
 
-export function BuyAssetUrl (network: Network, asset: AccountAssetOptionType, account: UserAccountType, buyAmount: string) {
-  switch (network) {
-    case Network.Mainnet:
+export function BuyAssetUrl (networkChainId: string, asset: AccountAssetOptionType, account: UserAccountType, buyAmount: string) {
+  switch (networkChainId) {
+    case '0x1':
```

Comment:
I'm concerned that we use hard coded chain ID string here.
We should somehow use shared value with C++, maybe through defining it in mojom.
Or if we can only define it in `brave_wallet_ui/constants/types.ts`, we should also put a comment to specify what list it should be syncing.

Reply:
done